test/samples/features/turnip_steps/global_steps.rb

```diff
@@ -0,0 +1,19 @@
+step %(a) do
+  sleep 0.10
+end
+
+step %(b) do
+  sleep 0.25
```

Comment:
Thank you for your review. Sorry, these `sleep`s were needless, I think.
I accidentally left these when I wrote this test referring cucumber test.
I removed these. (7a92f59)

I also removed other needless codes I found in turnip.bats/setup,teardown